_posts/2011-01-01-how-to-submit-apk.md

```diff
@@ -114,5 +114,13 @@ After a Poynt agent approves the Apk for the Poynt App Marketplace, status chang
 
 <img class="shadow image" src="../assets/AppSubmit_Screenshot29.png" alt="AppSubmit_Screenshot29" style="border:20px;margin:20px">
 
+
+### <a name="WhatsNew">What's New:</a>
```

Comment:
why not just update existing screenshot in the main flow, instead of adding 'what's new'?